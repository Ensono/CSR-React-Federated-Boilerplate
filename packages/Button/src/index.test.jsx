import React from "react"
import {render} from "@testing-library/react"
import {Button} from '.'

// jest.mock("../../utils/autocomplete", () => {
//     return {handleClick: jest.fn(), Focus: jest.fn(), handleKeyboardTapping: jest.fn()}
// })

describe("Components - Button: ", () => {
    let props = {type: 'primary'}

    it("should match the snapshot for type = primary", () => {
        const {asFragment} = render(
                <Button {...props} />
        )
        expect(asFragment()).toMatchSnapshot()
    })

    
})
