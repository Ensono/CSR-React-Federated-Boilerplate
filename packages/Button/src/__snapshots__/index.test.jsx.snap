// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Button:  should match the snapshot for type = primary 1`] = `
<DocumentFragment>
  <button
    style="margin-left: 10px; color: rgb(255, 255, 255); background-color: rgb(64, 158, 255); padding: 12px 20px; font-size: 14px; border-radius: 4px; outline: none; border: 1px solid #dcdfe6; cursor: pointer;"
  >
    primary Button
  </button>
</DocumentFragment>
`;
